example/hello_object_detection_ssd_node/main.js

```diff
@@ -0,0 +1,277 @@
+const ie = require('inference-engine-node');
+const jimp = require('jimp');
+const fs = require('fs').promises;
+const {performance} = require('perf_hooks');
+const {ObjectDetectionPrediction} = require('./box');
+
+const option_definitions = [
+  {
+    name: 'help',
+    alias: 'h',
+    type: Boolean,
+    description: 'Show this help message and exit.'
+  },
+  {
+    name: 'model',
+    alias: 'm',
+    type: String,
+    description: 'Required. Path to an .xml file with a trained model.'
+  },
+  {
+    name: 'image',
+    alias: 'i',
+    type: String,
+    description: 'Required. Path to an image file.'
+  },
+  {
+    name: 'device',
+    alias: 'd',
+    type: String,
+    defaultValue: 'CPU',
+    description:
+        'Optional. Specify the target device to infer on; CPU, GPU, FPGA, ' +
+        'HDDL, MYRIAD or HETERO: is acceptable. Default value is CPU.'
+  },
+  {
+    name: 'iterations',
+    alias: 'n',
+    type: Number,
+    defaultValue: 1,
+    description: 'Optional. The number of iterations for inference. ' +
+        'Default value is 1.'
+  },
+  {
+    name: 'threshold',
+    alias: 't',
+    type: Number,
+    defaultValue: 0.5,
+    // TODO: document
+    description: 'Optional. '
```

Comment:
Please keep consistent with README.md where it says "Optional. The minimal probability of the visibility object. Default value is 0.5."